meta-ibm/recipes-phosphor/webui/webui-vue/change-logo.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Please put the github issue this is resolving in the commit message.

Reply:
Done